src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -120,6 +138,18 @@ class editIndex extends viewModelBase {
                         <h2 class="margin-top margin-top-sm">${text}</h2>
                     </div>`;
         });
+
+        this.previewItem.extend({ rateLimit: 100});
```

Comment:
rate limit or throttle?

Reply:
1. from what I read, `throttle `is deprecated
2. I used `rateLimit` here similar to the preview item usage in query.ts
https://github.com/ravendb/ravendb/blob/1c050ea85f3de678e4f45880639ef10b52556b27/src/Raven.Studio/typescript/viewmodels/database/query/query.ts#L415